packages/reactotron-redux/src/create-action-tracker.js

```diff
@@ -39,8 +39,24 @@ export default (reactotron, trackerOptions = {}) => {
 
         var unwrappedAction = action.type === 'PERFORM_ACTION' && action.action ? action.action : action
 
+        // if matchException is true, actionType is matched with exception
+        const matchException = (exception, actionType) => {
+          if (typeof exception === 'string') {
+            return actionType === exception
+          } else if (typeof exception === 'function') {
+            return actionType === exception
+          } else if (exception instanceof RegExp) {
+            return exception.test(actionType)
+          } else {
+            return false
+          }
+        }
+
+        const matchExceptions = exceptions.reduce((prev, exception) => matchException(exception, unwrappedAction.type) || prev, false)
```

Comment:
Or even better, use `any` instead of `reduce`...

```
import { any } from 'ramda'
...
const matchExceptions = any(exception => matchException(exception, unwrappedAction.type), exceptions)
```